### ISSUE TITLE
Editor pause does not work

### ISSUE THREAD

**Issue by vonflyhighace2** [godotengine/godot#6765]
**Operating system or device - Godot version:**
windows 10 pro x64 
2.1 stable
**Issue description** (what happened, and what was expected):
Editor pause button does nothing.

**Steps to reproduce:**
press the pause button during testing a project.
 Is this still a feature or is it being phased out/ not supported?

**Link to minimal example project** (optional but very welcome):

**Comment by hetdegon** [comment]
Seems to work for me.

**Comment by memoryruins** [comment]
Pausing works only for the first time a scene is paused. If the scene is replayed, it no longer pauses until godot is restarted.

![pausing bug](https://i.imgur.com/1EvlVcD.gif)

Windows 10 x64
2.1.3

**Comment by memoryruins** [comment]
I receive this error when the pausing **works**.
Nothing goes to the console when it no longer works.
```
ERROR: Object::emit_signal: Error calling method from signal 'breaked': 'ScriptEditor::_breaked': Method expected 2 arguments, but called with 0.
   At: core\object.cpp:1162
```

**Comment by nunodonato** [comment]
I can confirm this also on Linux and Godot 3

**Comment by Noshyaar** [comment]
Can't reproduce anymore in 3.1 master 3ad9e47

**Comment by akien-mga** [comment]
Closing as fixed in 3.1 as per the above.
